### ISSUE TITLE
[BUG] Crashing sometimes in the StarrySky End Dimension

### ISSUE THREAD

**Issue by MSandro** [DaFuqs/StarrySky#25]
**Describe the bug**
Sometimes the game is crashing on travelin in the StarrySky End Dimension.
I have Better End installed.
Fabric 1.17.1

**To Reproduce**
Travel in the StarrySky End Dimension. I am not sure what exactly is causing the crash.

**Crash Report**
```
---- Minecraft Crash Report ----
// Quite honestly, I wouldn't worry myself about that.

Time: 9/22/21, 3:34 PM
Description: Feature placement

java.lang.ClassCastException: class net.minecraft.class_3233 cannot be cast to class net.minecraft.class_1937 (net.minecraft.class_3233 and net.minecraft.class_1937 are in unnamed module of loader net.fabricmc.loader.launch.knot.KnotClassLoader @141e5bef)
	at de.dafuqs.starrysky.dimension.spheroid.spheroids.Spheroid.placeSpawner(Spheroid.java:219)
	at de.dafuqs.starrysky.dimension.spheroid.spheroids.unique.EndCitySpheroid.placeShulkerSpawner(EndCitySpheroid.java:177)
	at de.dafuqs.starrysky.dimension.spheroid.spheroids.unique.EndCitySpheroid.decorate(EndCitySpheroid.java:113)
	at de.dafuqs.starrysky.dimension.SpheroidDecoratorFeature.method_13151(SpheroidDecoratorFeature.java:28)
	at net.minecraft.class_2975.method_12862(class_2975.java:58)
	at net.minecraft.class_2988.method_30384(class_2988.java:29)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
	at java.base/java.util.stream.IntPipeline$1$1.accept(IntPipeline.java:180)
	at java.base/java.util.stream.Streams$RangeIntSpliterator.forEachRemaining(Streams.java:104)
	at java.base/java.util.Spliterator$OfInt.forEachRemaining(Spliterator.java:699)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
	at net.minecraft.class_2988.method_13151(class_2988.java:28)
	at net.minecraft.class_2975.method_12862(class_2975.java:58)
	at net.minecraft.class_1959.method_8702(class_1959.java:388)
	at net.minecraft.class_2794.method_12102(class_2794.java:245)
	at net.minecraft.class_2806.method_12151(class_2806.java:100)
	at net.minecraft.class_2806.method_12154(class_2806.java:231)
	at net.minecraft.class_3898.method_17225(class_3898.java:556)
	at com.mojang.datafixers.util.Either$Left.map(Either.java:38)
	at net.minecraft.class_3898.method_17224(class_3898.java:550)
	at java.base/java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1146)
	at java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:478)
	at net.minecraft.class_3900.method_17634(class_3900.java:58)
	at net.minecraft.class_3846.method_16907(class_3846.java:91)
	at net.minecraft.class_3846.method_16900(class_3846.java:146)
	at net.minecraft.class_3846.run(class_3846.java:102)
	at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1434)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:295)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1016)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1665)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1598)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
	at de.dafuqs.starrysky.dimension.spheroid.spheroids.Spheroid.placeSpawner(Spheroid.java:219)
	at de.dafuqs.starrysky.dimension.spheroid.spheroids.unique.EndCitySpheroid.placeShulkerSpawner(EndCitySpheroid.java:177)
	at de.dafuqs.starrysky.dimension.spheroid.spheroids.unique.EndCitySpheroid.decorate(EndCitySpheroid.java:113)
	at de.dafuqs.starrysky.dimension.SpheroidDecoratorFeature.method_13151(SpheroidDecoratorFeature.java:28)
	at net.minecraft.class_2975.method_12862(class_2975.java:58)
	at net.minecraft.class_2988.method_30384(class_2988.java:29)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
	at java.base/java.util.stream.IntPipeline$1$1.accept(IntPipeline.java:180)
	at java.base/java.util.stream.Streams$RangeIntSpliterator.forEachRemaining(Streams.java:104)
	at java.base/java.util.Spliterator$OfInt.forEachRemaining(Spliterator.java:699)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)

-- Patchouli Book Info --
Details:
	Patchouli open book context: n/a
Stacktrace:
	at net.minecraft.class_128.handler$djn000$fillPatchouliContext(class_128.java:521)
	at net.minecraft.class_128.<init>(class_128.java:37)
	at net.minecraft.class_128.method_560(class_128.java:267)
	at net.minecraft.class_1959.method_8702(class_1959.java:390)
	at net.minecraft.class_2794.method_12102(class_2794.java:245)
	at net.minecraft.class_2806.method_12151(class_2806.java:100)
	at net.minecraft.class_2806.method_12154(class_2806.java:231)
	at net.minecraft.class_3898.method_17225(class_3898.java:556)
	at com.mojang.datafixers.util.Either$Left.map(Either.java:38)
	at net.minecraft.class_3898.method_17224(class_3898.java:550)
	at java.base/java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1146)
	at java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:478)
	at net.minecraft.class_3900.method_17634(class_3900.java:58)
	at net.minecraft.class_3846.method_16907(class_3846.java:91)
	at net.minecraft.class_3846.method_16900(class_3846.java:146)
	at net.minecraft.class_3846.run(class_3846.java:102)
	at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1434)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:295)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1016)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1665)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1598)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)

-- Feature --
Details:
	Description: ResourceKey[minecraft:worldgen/configured_feature / starry_sky:starry_sky_spheroid_decorator]

-- Generation --
Details:
	CenterX: 6
	CenterZ: 28
	Seed: -1231812840604318816
	Biome: net.minecraft.class_1959@27dbfcae

-- Chunk to be generated --
Details:
	Location: 6,28
	Position hash: 120259084294
	Generator: de.dafuqs.starrysky.dimension.StarrySkyChunkGenerator@43ce9e0

-- Affected level --
Details:
	All players: 1 total; [class_3222['MSandro'/375165, l='ServerLevel[1]', x=154.70, y=30.83, z=560.02]]
	Chunk stats: 1454
	Level dimension: starry_sky:starry_sky_end
	Derived: true
	Level spawn location: World: (8,72,8), Section: (at 8,8,8 in 0,4,0; chunk contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 138775 game time, 138775 day time
	Level name: 1
	Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: true
	Level weather: Rain time: 4390 (now: false), thunder time: 35654 (now: false)
	Known server brands: fabric
	Level was modded: true
	Level storage version: 0x04ABD - Anvil

-- System Details --
Details:
	Minecraft Version: 1.17.1
	Minecraft Version ID: 1.17.1
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 16.0.1, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 1449834360 bytes (1382 MiB) / 4294967296 bytes (4096 MiB) up to 4294967296 bytes (4096 MiB)
	CPUs: 16
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 7 2700X Eight-Core Processor         
	Identifier: AuthenticAMD Family 23 Model 8 Stepping 2
	Microarchitecture: Zen+
	Frequency (GHz): 3.70
	Number of physical packages: 1
	Number of physical CPUs: 8
	Number of logical CPUs: 16
	Graphics card #0 name: Radeon RX 580 Series
	Graphics card #0 vendor: Advanced Micro Devices, Inc. (0x1002)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x67df
	Graphics card #0 versionInfo: DriverVersion=27.20.15003.5017
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 2.13
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 8192.00
	Memory slot #1 clockSpeed (GHz): 2.13
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 27737.86
	Virtual memory used (MB): 22807.23
	Swap memory total (MB): 11406.74
	Swap memory used (MB): 244.38
	JVM Flags: 5 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx4096m -Xms256m -XX:PermSize=256m
	Fabric Mods: 
		adorn: Adorn 2.1.5+1.17.1
		advanced_runtime_resource_pack: Runtime Resource Pack 0.4.3
		advancements-enlarger: Advancements Enlarger 0.2.4
		agape_space: Space Dimensions 0.3.0
		alloy_forgery: Alloy Forgery 2.0.3
		amecsapi: Amecs API 1.1.4+mc21w16a
		appleskin: AppleSkin mc1.17-2.1.3
		architectury: Architectury 2.5.27
		audiooutput: AudioOutput 0.0.4
		authme: Auth Me 1.5.0
		auto_planter: AutoPlanter 17.1-fabric
		autoconfig1u: Auto Config v1 Updated 3.3.1
		automated_crafting: Automated Crafting 1.3.24+MC1.17-1.17.1
		awokenworld: Awoken World 1.1.0-beta
		balm-fabric: Balm 1.3.0+0
		bclib: BCLib 0.3.1
		betterend: Better End 0.11.0-pre
		betternether: Better Nether 5.1.3
		bitsandchisels: BitsAndChisels 2.5.3
		blockshifter: Blockshifter 1.4.0+fabric-1.17.1
		blockus: Blockus 2.2.10+1.17.1
		blue_endless_jankson: jankson 1.2.1
		breakprogress: Break Progress 1.0.1
		camp_fires_cook_mobs: Camp Fires Cook Mobs 1.2.1+MC1.17-1.17.1
		can-i-mine-this-block: cAn i MiNe thIS bLOCk? 1.1.0
		cardinal-components: Cardinal Components API 3.1.1
		cardinal-components-base: Cardinal Components API (base) 3.1.1
		cardinal-components-block: Cardinal Components API (blocks) 3.1.1
		cardinal-components-chunk: Cardinal Components API (chunks) 3.1.1
		cardinal-components-entity: Cardinal Components API (entities) 3.1.1
		cardinal-components-item: Cardinal Components API (items) 3.1.1
		cardinal-components-level: Cardinal Components API (world saves) 3.1.1
		cardinal-components-scoreboard: Cardinal Components API (scoreboard) 3.1.1
		cardinal-components-util: Cardinal Components API (utilities) 3.1.1
		cardinal-components-world: Cardinal Components API (worlds) 3.1.1
		carrier: Carrier 1.8.0
		charm: Charm 3.3.2
		chipped: Chipped 1.1.1
		cleanview: CleanView 1.17.1-v1-fabric
		cloth-api: Cloth API 2.0.54
		cloth-armor-api-v1: Cloth Armor API v1 1.5.47
		cloth-basic-math: Cloth Basic Math 0.5.1
		cloth-client-events-v0: Cloth Client Events v0 2.0.54
		cloth-common-events-v1: Cloth Common Events v1 2.0.54
		cloth-config2: Cloth Config v5 5.0.38
		cloth-datagen-api-v1: Cloth Datagen v1 2.0.54
		cloth-scissors-api-v1: Cloth Scissors API v1 2.0.54
		cloth-utils-v1: Cloth Utils v1 2.0.54
		clumps: Clumps 7.0.7
		coat: Coat 1.0.0-beta.9
		collective-fabric: Collective (Fabric) 1.40
		com_electronwill_night-config_core: core 3.6.3
		com_electronwill_night-config_toml: toml 3.6.3
		com_flowpowered_flow-math: flow-math 1.0.3
		com_github_dimensionaldevelopment_poly2tri_java: poly2tri.java 0.1.1
		com_moandjiezana_toml_toml4j: toml4j 0.7.2
		comforts: Comforts 0.0.6-1.17.1
		completeconfig: CompleteConfig 1.1.0
		compostable-rottenflesh: Compostable Rotten Flesh 1.0.0
		couplings: Couplings 1.5.0+1.17
		craftingtweaks: Crafting Tweaks 13.0.0+0
		crowdin-translate: CrowdinTranslate 1.3+1.17
		cullleaves: Cull Leaves 2.2.0
		customportalapi: Custom Portals 0.0.1-beta37-1.17
		danceofgrowth: Twerk / Crop Dusting Mod 1.0
		dankstorage: Dank Storage 2.1-1.17.1
		dark-enchanting: Dark Enchanting 0.4.1-1.17.1
		datapackportals: Datapack Portals 1.0.4-1.17
		datareload: Data Reload 1.1
		diggusmaximus: Diggus Maximus 1.4.5-1.17
		dimdoors: DimensionalDoors 4.0.0+beta.2-26.07.2021
		disable_custom_worlds_advice: Disable Custom Worlds Advice 1.3
		dml-refabricated: Deep Mob Learning: Refabricated 0.4.4-beta
		dml-refabricated-base: DML: Refabricated - Base 0.4.4-beta
		dml-refabricated-events: DML: Refabricated - Vanilla Events 0.4.4-beta
		dml-refabricated-modular-armor: DML: Refabricated - Glitch Armor 0.4.4-beta
		dml-refabricated-rei: DML: Refabricated - REI Integration 0.4.4-beta
		drippyloadingscreen: Drippy Loading Screen 1.1.1
		dynamicfps: Dynamic FPS 2.0.5
		enhancedblockentities: Enhanced Block Entities 0.5+1.17
		entityculling: EntityCulling-Fabric 1.3.3
		expandedstorage: Expanded Storage 7.1.2
		extraalchemy: Extra Alchemy 1.6.1
		extragenerators: Extra Generators 1.1-BETA+1.17
		fabric: Fabric API 0.40.1+1.17
		fabric-api-base: Fabric API Base 0.3.0+a02b446313
		fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.3.0+cbda931818
		fabric-biome-api-v1: Fabric Biome API (v1) 3.2.0+cbda931818
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.5+a02b446318
		fabric-command-api-v1: Fabric Command API (v1) 1.1.3+5ab9934c18
		fabric-commands-v0: Fabric Commands (v0) 0.2.2+92519afa18
		fabric-containers-v0: Fabric Containers (v0) 0.1.12+cbda931818
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.3.0+cbda931818
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.5+be9da31018
		fabric-dimensions-v1: Fabric Dimensions API (v1) 2.0.11+6cefd57718
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.2.3+87cc6e4c18
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.10+fc40aa9d18
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.1+92519afa18
		fabric-furnaces: Fabric Furnaces 2.0.1-1.17
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.7+cbda931818
		fabric-item-api-v1: Fabric Item API (v1) 1.2.4+a02b44633d
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.2.10+b7ab612118
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.4+cbda931818
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.2+36b77c3e18
		fabric-language-kotlin: Fabric Language Kotlin 1.6.4+kotlin.1.5.30
		fabric-language-scala: Fabric Language Scala 1.1.0+scala.2.13.6
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.4.4+a02b44633d
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.4+a02b446318
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.3+92519afa18
		fabric-models-v0: Fabric Models (v0) 0.3.0+a02b446318
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.13+cbda931818
		fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.11+a02b446318
		fabric-networking-v0: Fabric Networking (v0) 0.3.2+92519afa13
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.10.9+cbda931818
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.3+a02b446318
		fabric-particles-v1: Fabric Particles (v1) 0.2.4+a02b446318
		fabric-permissions-api-v0: fabric-permissions-api 0.1-SNAPSHOT
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.7.10+e2961fee18
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.4.4+cbda931818
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.8+cbda931818
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.3+4658223018
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.5+a02b446318
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.14+4658223018
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.4+4658223018
		fabric-rendering-v1: Fabric Rendering (v1) 1.8.2+ffb6d41e18
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.4.8+a00e834b18
		fabric-screen-api-v1: Fabric Screen API (v1) 1.0.4+198a96213d
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.8+cbda931818
		fabric-structure-api-v1: Fabric Structure API (v1) 1.1.13+5ab9934c18
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.2.1+b06cb95b18
		fabric-textures-v0: Fabric Textures (v0) 1.0.6+a02b446318
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.2.12+b7ab61213d
		fabric-transfer-api-v1: Fabric Transfer API (v1) 1.3.1+4658223018
		fabricloader: Fabric Loader 0.11.6
		fake-player-api: Fake Player API 0.2.0
		farmersdelight: Farmer's Delight 1.17.1-0.1.2
		fastchest: FastChest 1.2+1.17
		fastfurnace: FastFurnace 1.17-3.0
		fasttransferlib: Fast Transfer Lib 0.5.6
		ferritecore: FerriteCore 3.0.3
		fiber: fiber 0.23.0-2
		forgottenrecipes: Forgotten Recipes 1.0.3
		gateofbabylon: Gate Of Babylon 1.5.1-1.17.1
		gearreborn: Gear Reborn 1.17-0.1.10
		glassential: Glassential 1.17-1.2.0
		golemsgalore: Golems Galore 1.0.3-1.17.1
		grass_pass: Grass Bypass 1.0.0
		grindenchantments: Grind Enchantments 1.4.0+1.17.1
		hardcorequesting: Hardcore Questing Mode 1.17.1-5.6.2
		hctm-base: HCTM Base 3.6.2
		hologram-api: Hologram API 0.2.1+1.17.1
		iknowwhatimdoing: I Know What I'm Doing 3.1.0
		improved-stations: Improved Stations 2.0.7
		indrev: Industrial Revolution 1.10.0-alpha.12
		inmis: Inmis 2.3.2-1.17.1
		jankson: Jankson 3.0.1+j1.2.0
		java: OpenJDK 64-Bit Server VM 16
		justenoughkeys: Just Enough Keys 0.2.2
		kanos_config: Kanos Config 0.1.4+1.14.4-1.17.1
		kibe: Kibe 1.9.1-BETA+1.17
		kjsextras: KubeJS Extras 1.3.2
		konkrete: Konkrete 1.2.3
		ktdatataglib: KtDataTagLib 1.5.4
		kubejs: KubeJS 1605.3.10-build.9999
		kyrptconfig: Kytpt Config 1.1.10-1.17
		lambdynlights: LambDynamicLights 2.0.2+1.17
		lapisreserve: Lapis Reserve 1.0.8
		lazydfu: LazyDFU 0.1.2
		lchunkloader: Literally Chunk Loader 2.0.3
		libaddict: libaddict 0.3.0
		libblockattributes: LibBlockAttributes 0.9.0
		libblockattributes_core: LibBlockAttributes (Core) 0.9.0
		libblockattributes_fluids: LibBlockAttributes (Fluids) 0.9.0
		libblockattributes_items: LibBlockAttributes (Items) 0.9.0
		libcd: LibCapableData 3.0.3+1.16.3
		libgui: LibGui 4.1.7+1.17.1
		libmultipart: LibMultiPart 0.6.0
		libnetworkstack: Lib Network Stack 0.5.0
		libninepatch: LibNinePatch 1.1.0
		libzoomer: LibZoomer 0.2.1+1.17
		lifts: Lifts 1.3-BETA+1.17
		light-fix: light-fix 1.0.0
		lithium: Lithium 0.7.4
		locki: Locki 0.5.0
		magna: Magna 1.5.5-1.17-pre4
		matrix: Matrix 1.0.0
		mcskyblock: MC Skyblock Fabric 1.0.2
		megane: megane 5.4.1
		megane-base: megane-base 5.4.1
		megane-extra-generators: megane-extra-generators 5.4.1+1.1-BETA-1.17
		megane-fabric-transfer: megane-fabric-transfer 5.4.1+1.3.0-fc40aa9d18
		megane-fast-transfer-lib: megane-fast-transfer-lib 5.4.1+0.5.7
		megane-industrial-revolution: megane-industrial-revolution 5.4.1+1.10.0-alpha.12
		megane-kibe: megane-kibe 5.4.1+1.9.1-BETA-1.17
		megane-lib-block-attributes: megane-lib-block-attributes 5.4.1+0.9.0
		megane-modern-industrialization: megane-modern-industrialization 5.4.1+0.5.12
		megane-reborn-core: megane-reborn-core 5.4.1+5.0.8-beta-build.111
		megane-runtime: megane-runtime 5.4.1
		megane-team-reborn-energy: megane-team-reborn-energy 5.4.1+2.0.0-beta1
		megane-tech-reborn: megane-tech-reborn 5.4.1+5.0.8-beta-build.111
		megane-vanilla: megane-vanilla 5.4.1+1.17.1
		megane-wireless-networks: megane-wireless-networks 5.4.1+1.1.2
		minecraft: Minecraft 1.17.1
		mm: Manningham Mills 2.1
		mo_glass: Mo Glass 1.5-MC1.17.1
		modern_industrialization: Modern Industrialization 0.5.12
		modmenu: Mod Menu 2.0.12
		moss_fix: Moss Fix 1.1.0
		mousewheelie: Mouse Wheelie 1.7.3-newconfig+mc1.17.1-pre1
		myron: Myron 1.6.1
		no-potion-offset: No Potion Offset ${version}
		notes: Notes 1.0.6
		okzoomer: Ok Zoomer 5.0.0-beta.2+1.17
		omega-config: OmegaConfig 1.0.8
		org_aperlambda_lambdajcommon: lambdajcommon 1.8.1
		org_codehaus_groovy_groovy: groovy 3.0.8
		org_codehaus_groovy_groovy-jsr223: groovy-jsr223 3.0.8
		org_jetbrains_annotations: annotations 19.0.0
		org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.5.30
		org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.5.30
		org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.5.30
		org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.5.30
		org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.5.1
		org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.5.1
		org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.2.2
		org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.2.2
		org_jgrapht_jgrapht-core: jgrapht-core 1.1.0
		org_objenesis_objenesis: objenesis 3.2
		owo: OwO 0.1
		packages: Packages 1.2.1
		packet_tweaker: Packet Tweaker 0.2.0-1.17-pre1
		paintingcycle: Painting Cycle 1.0.0
		paradoxslabs: ParadoxSlabs 0.1.1
		patchouli: Patchouli 1.17.1-55-FABRIC-SNAPSHOT
		pehkui: Pehkui 2.4.0+1.14.4-1.17.1
		placeholder-api: Placeholder API 1.1.0+1.17.1
		plantinajar: Plant In A Jar 2.2.2
		playerabilitylib: Pal 1.3.0
		pling: Pling 1.4.0
		polymer: Polymer 0.1.0-rc.4+1.17.1
		pugh_lib: Pugh Lib ${version}
		qcommon-cfg-core: qcommon-cfg-core 3.3.0
		qcommon-croco: Croco 2.1.1
		reach-entity-attributes: Reach Entity Attributes 2.1.1
		reborncore: Reborn Core 5.0.8-beta+build.111
		redbits: Red Bits 1.10.0
		replanter: Replanter 1.3
		resourceful_tools: Resourceful Tools 1.1.20
		rhino: Rhino 1605.1.2-build.40
		roughlyenoughitems: Roughly Enough Items 6.0.279-alpha
		rswires: RSWires 4.11.2
		scf: Super Cobblestone Funnels 2.1.0
		schmucks: Schmucks 1.4.0
		seasons: Fabric Seasons 1.2.2-BETA+1.17
		seedy-behavior: Seedy Behavior 1.0.1
		selfgoverningcatalyst: Self-Governing Catalyst 1.17-1.0.7
		server_translations_api: Server Translations API 1.4.5+1.17
		sgui: SGui 1.0.0-rc4+1.17.1
		simple_pipes: SimplePipeTest 0.6.1
		simple_pipes_dep_container: SimplePipeTest (Dependency Container) 0.6.1
		simpletech: SimpleTech 0.0.6
		skyutils: Sky utilities Mod 1.0.5_beta4
		slight-gui-modifications: 'Slight' GUI Modifications 2.0.1
		slotlink: slotlink 3.1.1
		smallernetherportals-fabric: Smaller Nether Portals (Fabric) 1.8
		someforgepatchesported: Some Forge Patches Ported 1.0.1
		somnus: Somnus API 0.0.13-1.17.1
		spatialharvesters: Spatial Harvesters 0.11.12a
		spruceui: SpruceUI 3.2.0+1.17
		stacc: Stacc 1.2.3
		starry_sky: Starry Skies 1.1.0-1.17.0
		steedstopper: Steed Stopper 1.0.1
		step-height-entity-attribute: Step Height Entity Attribute 1.0.0
		storagecabinet: Storage Cabinet 1.17.1-1.3.1
		structureworld: Structure World 1.1.1-BETA+1.17
		structurized: Structurized 1.3.0
		tatters: Tatters 1.1.0+1.17
		team_reborn_energy: Energy 2.0.0-beta1
		techreborn: Tech Reborn 5.0.8-beta+build.111
		terraform-wood-api-v1: Terraform Wood API (v1) 2.0.2
		terrarianslimes: Terrarian Slimes 1.1-BETA+1.17
		the_aether: The Aether 1.17.1-1.5.0
		things: Things 0.2.5
		timeinabottle: Time in a Bottle 1.6-mc1.17-fabric
		torcherino: Torcherino 15.0.0
		totemmod: Wither Totems Mod 0.3.2
		trinkets: Trinkets 3.0.4
		truerooms: True Secret Rooms 1.1.5
		tweed4_annotated: tweed4_annotated 1.0.0
		tweed4_base: tweed4_base 1.0.0
		tweed4_data: tweed4_data 1.0.0
		tweed4_data_hjson: tweed4_data_hjson 1.0.0
		tweed4_tailor_coat: tweed4_tailor_coat 1.0.0
		tweed4_tailor_screen: tweed4_tailor_screen 1.0.0
		universal-graves: Universal Graves 1.0.8+1.17.1
		wands: Building Wands 2.2.3
		waystones: Waystones 2.0.4
		wirelessnetworks: Wireless Networks 1.1.2
		workings: Workings 1.0.7
		wthit: wthit 3.9.1
		xaerominimap: Xaero's Minimap 21.16.0
		xaeroworldmap: Xaero's World Map 1.17.0
	Player Count: 1 / 8; [class_3222['MSandro'/375165, l='ServerLevel[1]', x=154.70, y=30.83, z=560.02]]
	Data Packs: vanilla, Fabric Mods, dimdoors/classic, dimdoors/default
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'fabric`
```

**Comment by DaFuqs** [comment]
The crash happened, because an end city sphere could not place a shulker spawner. 
Does this happen frequently or just very sporadically? Any mods that modify mob spawners?

Adding compatibility or a workaround will be an easy fix.

**Comment by MSandro** [comment]
Yes it crashes very often when I enter some areas in the dimension. No, as I know there is no mod installed which causing mob spawn changes.

**Comment by DaFuqs** [comment]
Fixed in 1.2.0
